charts/opentelemetry-operator/values.schema.json

```diff
@@ -685,48 +689,45 @@
                         "Fail"
                     ]
                 },
-                "enabled": {
+                "create": {
                     "type": "boolean",
                     "default": false,
-                    "title": "The enabled Schema",
+                    "title": "The create Schema",
                     "examples": [
                         true
                     ]
                 },
+                "secretName": {
+                    "type": "string",
+                    "default": "",
+                    "title": "The secretName Schema",
+                    "examples": [
+                        ""
+                    ]
+                },
                 "timeoutSeconds": {
                     "type": "integer",
-                    "default": 10,
+                    "default": 0,
                     "title": "The timeoutSeconds Schema",
                     "examples": [
-                      10
+                        10
                     ]
                 },
                 "namespaceSelector": {
                     "type": "object",
                     "default": {},
                     "title": "The namespaceSelector Schema",
-                    "examples": [{
-                        "matchExpressions": [
-                            {
-                                "key": "runlevel",
-                                "operator": "NotIn",
-                                "values": [
-                                    "0",
-                                    "1"
-                                ]
-                            }
-                        ]
-                    }]
+                    "required": [],
+                    "properties": {},
+                    "examples": [{}]
                 },
                 "objectSelector": {
                     "type": "object",
                     "default": {},
                     "title": "The objectSelector Schema",
-                    "examples": [{
-                        "matchLabels": {
-                            "foo": "bar"
-                        }
-                    }]
+                    "required": [],
+                    "properties": {},
+                    "examples": [{}]
```

Comment:
Why example are removed from here and above? seems like an unrelated change

Reply:
regenerated the schema based on current values.

Reply:
I mean `"examples": [{}]` is pretty useless now :)

Reply:
oh, it's pretty late here :) Maybe @Sadzeih  manually added them in the json

Reply:
@dmitryax we can probably address this in another pr , my schema is generated based on the current values. Can we merge ? Tomorrow is Monday :)